package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Was this file created by accidentally running npm install in the root dir? Looks like we can safely discard to reduce clutter

Reply:
I don't know how it got there - you're probably right - but it's gone now - thanks!